subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/JavaCompile.java

```diff
@@ -90,6 +91,7 @@
  *     }
  * </pre>
  */
+@SuppressWarnings("MethodMayBeStatic")
```

Comment:
:x: Don't suppress warnings at class level for a single method affected. Also, why not make it static?